docs/docs/cli/generated/tracetest.md

```diff
@@ -0,0 +1,31 @@
+# CLI Referece
+## tracetest
```

Comment:
Is there a way to make the generator name `tracetest` with the first letter in uppercase? (`Tracetest`)

Reply:
I couldn't find anything for that. This happens because this subtitle is the CLI command.

Reply:
In that case we can keep it as it is for now